### PR TITLE
Add admin logging forcemap command

### LINKED ISSUE #28789: forcemap command should be logged
also should shown in achat, like with gamerules

### PR DESCRIPTION
**Pull request by lzk228**
## About the PR

- Added logging for forcemap command
- Also notification in achat

## Why / Balance

Useful feature
Fixes #28789 

## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
ADMIN:
- add: Added logging for forcemap command.

### CODE REVIEW

**Comment by ShadowCommander** [comment]
@Chief-Engineer check the message text on this.

File: Content.Server/GameTicking/Commands/ForceMapCommand.cs [hunk 1]
```diff
@@ -29,6 +36,16 @@ public void Execute(IConsoleShell shell, string argStr, string[] args)
             var gameMap = IoCManager.Resolve<IGameMapManager>();
             var name = args[0];
 
+            if (shell.Player != null)
+            {
+                _adminLogger.Add(LogType.AdminMessage, $"{shell.Player} tried to forcemap {name} via command");
+                _chatManager.SendAdminAnnouncement(Loc.GetString("forcemap-command-admin-notification", ("name", name), ("admin", shell.Player)));
```
[the remark this answers is not in the document]
Comment (lzk228) [reply Content.Server/GameTicking/Commands/ForceMapCommand.cs:42]: > Also the AdminMessage log type is for admin messages to players like prayers

i couldnt found which type should i use here

**Comment by lzk228** [comment]
oops

**Comment by lzk228** [comment]
closing because logging for such commans should be more general, and not specified like i did
as was discussed in discord with Chief_Engineer https://discord.com/channels/310555209753690112/560845886263918612/1251888799189827734

**Pull request closed without merging**